draft-irtf-cfrg-vdaf.md

```diff
@@ -1661,32 +1661,38 @@ def expand_into_vec(Prg,
 ### PrgSha3 {#prg-sha3}
 
 This section describes PrgSha3, a PRG based on the Keccak permutation of SHA-3
-{{FIPS202}}. Keccak is used in the cSHAKE128 mode of operation {{SP800-185}}.
+{{FIPS202}}. Keccak is used in the SHAKE128 mode of operation {{FIPS202}}.
 This Prg is RECOMMENDED for all use cases within VDAFs.
 
 ~~~
 class PrgSha3(Prg):
-    """PRG based on SHA-3 (cSHAKE128)."""
+    """PRG based on SHA-3 (SHAKE128)."""
 
     # Associated parameters
     SEED_SIZE = 16
 
     def __init__(self, seed, dst, binder):
+        # Implementation note: it is required that the length of dst be
+        # at most 255 bytes. This comes from the use of cSHAKE128's padding
+        # scheme (see {{SP800-185}} for details).
```

Comment:
This text needs to be normative, so I think we should move it out of a code comment and into its own paragraph. See the suggestion above.

Similarly for PrgFixedKeyAes128 below.